common/views/components/BasePage/BasePage.js

```diff
@@ -37,43 +39,64 @@ const BasePage = ({
   return (
     <PageBackgroundContext.Provider value={isCreamy ? 'cream' : 'white'}>
       <article data-wio-id={id}>
-        <Fragment>{Header}</Fragment>
+        <SpacingSection>{Header}</SpacingSection>
         <div className={classNames({
-          'bg-cream': isCreamy,
-          [spacing({s: 6}, {padding: ['bottom']})]: true
+          'bg-cream': isCreamy
         })}>
-          <div className='basic-page'>
-            <Fragment>{Body}</Fragment>
-          </div>
+          <SpacingSection>
+            <div className='basic-page'>
+              <Fragment>{Body}</Fragment>
+            </div>
+          </SpacingSection>
 
           {children &&
-            <Layout8>
-              {children}
-            </Layout8>
+            <SpacingSection>
+              {Children.map(children, (child, i) => (
+                <Fragment>
+                  {child &&
+                    <SpacingComponent>
+                      <Layout8>
+                        {child}
+                      </Layout8>
+                    </SpacingComponent>
+                  }
+                </Fragment>
+              ))}
+            </SpacingSection>
           }
 
           {contributorProps && contributorProps.contributors.length > 0 &&
-            <Layout8>
-              <div className={`${spacing({s: 6}, {margin: ['top']})}`}>
+            <SpacingSection>
+              <Layout8>
                 <Contributors {...contributorProps} />
-              </div>
-            </Layout8>
+              </Layout8>
+            </SpacingSection>
           }
 
           {Siblings.length > 0 &&
-            <Layout8>
+            <SpacingSection>
               {Children.map(Siblings, (child, i) => (
-                <div className={`${spacing({s: 6}, {margin: ['top']})}`}>
-                  {Siblings}
-                </div>
+                <Fragment>
+                  {child &&
+                    <SpacingComponent>
```

Comment:
Is this now a spacing component in a spacing component?
And is that fine?

Reply:
It's a `SpacingComponent` in a `SpacingSection` – and I think that's fine, nay, encouraged!